### ISSUE TITLE
How do keras calculate the loss in fit() or evaluate() functions?

### ISSUE THREAD

**Issue by curry111** [fchollet/keras#6292]
i think it is not the loss per epoch, 
but does it calculate the loss per mini-batch divided by the batch size?
and what about the evaluate() function?
- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
